chevah/compat/nt_users.py

```diff
@@ -193,10 +193,13 @@ def userExists(self, username):
             error_text = u'[%s] %s %s' % (number, name, message)
             self.raiseFailedtoCheckUserExists(username, error_text)
 
-    def isUserInGroups(self, username, groups, token):
+    def getGroupForUser(self, username, groups, token):
         """
-        Return true if `username` is a member of `groups`.
+        See: IOSUsers
         """
+        if not groups:
+            raise ValueError('Groups for validation can\'t be empty.')
```

Comment:
This is the new change since previous review.

I don't know if we should just remove this code and have None returned on empty list + and error when groups is None.

Reply:
What I would generally expect would be the set theory interpretation (if you remember my JS test about a set having the empty set as a subset).

But the question is, if there are no allowed groups, should we deny access to everyone? In that case, it seems to me a misconfiguration, and it would be hard to debug. But this special ValueError shows us (and possibly the users if they read logs) where to look.

While this introduces a slight inconsistency according to set theory, it would definitely aid in debugging IF we ever call the method with no groups. So, I think this is practical.